modules/nw-network-observability-operator.adoc

```diff
@@ -3,13 +3,13 @@
 
 :_content-type: PROCEDURE
 [id="nw-network-observability-operator_{context}"]
-= Network Observability Operator
+= Viewing statuses
 
-When you deploy flow collector, the Operator provides dashboards, metrics, and keeps flows accessible in the Grafana Loki log store. When a flow collector is deployed, new dashboards are available in the {product-title} console.
+The Network Observability Operator implements the Flow Collector API resource. The Flow Collector is created at the time of installation. When you deploy the Flow Collector, the Operator provides dashboards, metrics, and keeps flows accessible in the Grafana Loki log store. When a Flow Collector is deployed, new dashboards are available in the {product-title} console.
```

Comment:
It seems like the two last sentences are repeating the same information?

Reply:
Yea, let me consolidate those points a little. Good catch.